src/main/java/teammates/ui/controller/InstructorFeedbackEditSaveAction.java

```diff
@@ -52,11 +52,13 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                     + "<br><span class=\"bold\">Session visible from:</span> " + feedbackSession.sessionVisibleFromTime
                     + "<br><span class=\"bold\">Results visible from:</span> " + feedbackSession.resultsVisibleFromTime
                     + "<br><br><span class=\"bold\">Instructions:</span> " + feedbackSession.instructions;
+            data.setHasError(false);
         } catch (InvalidParametersException e) {
             setStatusForException(e);
+            data.setHasError(true);
         }
-        
-        return createRedirectResult(data.getInstructorFeedbackEditLink(courseId, feedbackSessionName));
+        data.setStatusForAjax(statusToUser.get(0));
```

Comment:
Only displaying the first <code>statusToUser</code> may not be good? We should be able to handle displaying multiple status messages. If the user has and is using multiple tabs on teammates, sometimes the statusMessage may show up in the wrong tab. The code in this action may also in future add multiple status messages.